source/simulator/core.cc

```diff
@@ -467,17 +471,10 @@ namespace aspect
                              "has not been tested in non-Cartesian geometries and currently requires "
                              "the use of a Cartesian geometry model."));
 
-    for (const auto &p : parameters.prescribed_traction_boundary_indicators)
-      boundary_traction[p.first]
-        = BoundaryTraction::create_boundary_traction<dim> (p.second.second);
-
-    for (auto &bv : boundary_traction)
-      {
-        if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(bv.second.get()))
-          sim->initialize_simulator(*this);
-        bv.second->parse_parameters (prm);
-        bv.second->initialize ();
-      }
+    std::set<types::boundary_id> open_traction_boundary_indicators
+      = geometry_model->get_used_boundary_indicators();
+    for (const auto &p : boundary_traction_manager.get_active_boundary_traction_names())
+      open_traction_boundary_indicators.erase (p.first);
```

Comment:
This part is not used and not necessary for boundary traction plugins. Remove these green lines (474-477).